bitbots_localization_handler/bitbots_localization_handler/localization_dsd/localization_blackboard.py

```diff
@@ -3,20 +3,22 @@
 from bitbots_blackboard.capsules.game_status_capsule import GameStatusCapsule
 from rclpy.duration import Duration
 from bitbots_localization.srv import ResetFilter, SetPaused
-
+from rclpy.node import Node
 
 class LocalizationBlackboard:
 
-    def __init__(self, node):
+    def __init__(self, node:Node):
         self.node = node
 
         self.shut_down_request = False
         self.last_initialized = None
         self.initialized = False
+        self.use_sim_time = self.node.get_parameter('use_sim_time').value
```

Comment:
does value with out a type work out of the box?

Reply:
Yes.